packages/react-core/src/components/TextInput/TextInput.tsx

```diff
@@ -193,6 +201,9 @@ export class TextInputBase extends React.Component<TextInputProps, TextInputStat
     } = this.props;
 
     const hasStatusIcon = ['success', 'error', 'warning'].includes(validated);
+    const ariaExpandedProps = expandedProps?.isExpanded
+      ? { 'aria-expanded': expandedProps?.isExpanded, 'aria-controls': expandedProps?.ariaControls, role: 'combobox' }
+      : {};
```

Comment:
```suggestion
    const ariaExpandedProps = expandedProps
      ? { 'aria-expanded': expandedProps.isExpanded, 'aria-controls': expandedProps.ariaControls, role: 'combobox' }
      : {};
```

We'll want these props added as long as expandedProps is passed rather than only when `isExpanded` is true.